packages/apps/composer-app/vite.config.ts

```diff
@@ -88,7 +88,11 @@ export default defineConfig({
       content: [
         resolve(__dirname, './index.html'),
         resolve(__dirname, './src/**/*.{js,ts,jsx,tsx}'),
-        resolve(__dirname, '../plugins/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../experimental/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../plugins/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../plugins/experimental/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../sdk/*/src/**/*.{js,ts,jsx,tsx}'),
+        resolve(__dirname, '../../ui/*/src/**/*.{js,ts,jsx,tsx}'),
```

Comment:
iirc the theme plugin itself has a place where these are listed so as not to need to repeat them for every app